jscomp/frontend/ppx_entry.ml

```diff
@@ -30,12 +30,12 @@ let rewrite_signature (ast : Parsetree.signature) : Parsetree.signature =
   let ast =
     match !Js_config.jsx_version with
     | None -> ast
-    | Some jsxVersion ->
+    | Some jsx_version_ ->
```

Comment:
Manual intervention here due to a shading error.

![image](https://github.com/rescript-lang/rescript-compiler/assets/16160544/47478689-47e8-48e8-950b-d3d37e670011)